src/modelinstance.cpp

```diff
@@ -190,13 +190,65 @@ Status ModelInstance::loadInputTensors(const ModelConfig& config, const DynamicM
 
         auto mappingName = config.getMappingInputByKey(name);
         auto tensor = std::make_shared<TensorInfo>(name, mappingName, precision, shape, layout);
-        this->inputsInfo[tensor->getMappedName()] = std::move(tensor);
+        // this->inputsInfo[tensor->getMappedName()] = std::move(tensor);
     }
-    SPDLOG_INFO("Final network inputs: {}", getNetworkInputsInfoString(networkInputs, config));
+    auto inputTensors = this->network_2->inputs();
+    for (const auto& input : inputTensors) {
+        std::string name;
+        try {
+            name = input.get_any_name();
+        } catch (const ov::Exception& e) {
+            SPDLOG_LOGGER_ERROR(modelmanager_logger, "Failed to get input name for model:{}; version:{}; from OpenVINO with error:{}",
+                getName(),
+                getVersion(),
+                e.what());
+            // TODO potentially allow for empty names if OV will load such model. Then potentially use empty string as input/output names
+            // and adjust validation, metadata, dags for that
+            return StatusCode::UNKNOWN_ERROR;
+        } catch (const std::exception& e) {
+            SPDLOG_LOGGER_ERROR(modelmanager_logger, "Failed to get input name for model:{}; version:{}; from OpenVINO with error:{}",
+                getName(),
+                getVersion(),
+                e.what());
+            return StatusCode::UNKNOWN_ERROR;
+        } catch (...) {
+            SPDLOG_LOGGER_ERROR(modelmanager_logger, "Failed to get input name for model:{}; version:{}; from OpenVINO",
+                getName(),
+                getVersion());
+            return StatusCode::UNKNOWN_ERROR;
+        }
+        auto precision = ovElementTypeToOvmsPrecision(input.get_element_type());
+        // TODO auto layout = input->getLayout();
+        /*
+        if (config.getLayouts().size() > 0) {
+            auto mappedName = config.getMappingOutputByKey(name);
+            auto it = config.getLayouts().find(mappedName == "" ? name : mappedName);
+            if (it != config.getLayouts().end()) {
+                layout = TensorInfo::getLayoutFromString(it->second);
+            }
+        }
+        input->setLayout(layout);
+        */
+        auto OVshape = input.get_shape();
```

Comment:
OpenVINO has `get_partial_shape` as well. We should test what happens if we load model with dynamic shape and catch possible exception during model loading. `get_shape` might throw an exception when shape is dynamic.